app/assets/stylesheets/shared/header_menu.scss

```diff
@@ -0,0 +1,12 @@
+$nav-background: #ab0404;
+$nav-border: 1px solid rgb(75, 75, 75);
```

Comment:
Let's try to be more descriptive with these variable names since they could be anything, not necessary only colors. What do you think about: `$nav-bg-color` and `$nav-border-color`?

Reply:
Also, is OK to leave them here for now since they are only a couple, but in the future, bear in mind to create an exclusive stylesheet for them, like `stylesheets/abstracts/_variables.scss` 😬

Reply:
Alright! Will keep it in mind